Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -402,190 +405,190 @@ class CallController: @unchecked Sendable {
         ring: Bool,
         migratingFrom: String?
     ) async throws {
-        if let migratingFrom {
-            executeOnMain { [weak self] in
-                self?.call?.state.reconnectionStatus = .migrating
-            }
-            webRTCClient?.prepareForMigration(
-                url: response.credentials.server.url,
-                token: response.credentials.token,
-                webSocketURL: response.credentials.server.wsEndpoint,
-                fromSfuName: migratingFrom
-            )
-        } else {
-            webRTCClient = environment.webRTCBuilder(
-                user,
-                apiKey,
-                response.credentials.server.url,
-                response.credentials.server.wsEndpoint,
-                response.credentials.token,
-                callCid(from: callId, callType: callType),
-                sessionID,
-                response.ownCapabilities,
-                videoConfig,
-                response.call.settings.audio,
-                .init()
-            )
-            webRTCClient?.onSignalConnectionStateChange = { [weak self] state in
-                self?.handleSignalChannelConnectionStateChange(state)
-            }
-            webRTCClient?.onSessionMigrationEvent = { [weak self] in
-                self?.handleSessionMigrationEvent()
-            }
-            webRTCClient?.onSessionMigrationCompleted = { [weak self] in
-                self?.call?.update(reconnectionStatus: .connected)
-            }
-        }
-        
+//        if let migratingFrom {
```

Comment:
is this a WIP?

Reply:
I only kept it in to help us identify if I missed something.